brave/browser/brave_browser_context.cc

```diff
@@ -162,6 +172,21 @@ BraveBrowserContext::BraveBrowserContext(
         atom::AtomBrowserContext::From(parent_partition, false));
   }
 
+  bool isolated_storage;
+  if (options.GetBoolean("isolated_storage", &isolated_storage)) {
+    isolated_storage_ = isolated_storage;
+  }
+
+  std::string tor_proxy;
+  if (options.GetString("tor_proxy", &tor_proxy)) {
+    tor_proxy_ = GURL(tor_proxy);
+    if (!tor_process_.IsValid()) {
+      base::FilePath tor("/usr/local/bin/tor");
```

Comment:
@jumde this path need to be changed depends on the binary location we want to bundle

Reply:
i would slightly prefer that this be `./app/extensions/bin/tor` (relative to root of `browser-laptop`) similar to what we will do with geth: https://github.com/brave/browser-laptop/pull/13177/files#diff-a108ee634057ec3892dd3354639d2117R11. 

see `getExtensionsPath` in browser-laptop for how to get this path

Reply:
that's better. we can export a option `tor_bin_path` and passing it from browser-laptop

Reply:
and we can have a unified path among different platforms